### PR TITLE
srtp: added sending of MENC_EVENT_SECURE event

### PR DESCRIPTION
**Pull request by juha-h**

### CODE REVIEW

File: modules/srtp/srtp.c [hunk 1]
```diff
@@ -281,6 +289,17 @@ static int start_crypto(struct menc_st *st, const struct pl *key_info)
 	info("srtp: %s: SRTP is Enabled (cryptosuite=%s)\n",
 	     sdp_media_name(st->sdpm), st->crypto_suite);
 
+	if (st->sess->eventh) {
+		if (re_snprintf(buf, sizeof(buf), "%s,%s",
+				sdp_media_name(st->sdpm),
+				st->crypto_suite))
+			(st->sess->eventh)(MENC_EVENT_SECURE, buf,
```
Comment (alfredh) [inline modules/srtp/srtp.c]: I think the paranthesis around st->sess->eventh can be removed...

**Comment by alfredh** [comment]
@juha-h thanks, this looks good. just a small comment.

**Comment by juha-h** [comment]
Alfred E. Heggestad writes:

> > @@ -281,6 +289,17 @@ static int start_crypto(struct menc_st *st, const struct pl *key_info)
>  	info("srtp: %s: SRTP is Enabled (cryptosuite=%s)\n",
>  	     sdp_media_name(st->sdpm), st->crypto_suite);
>  
> +	if (st->sess->eventh) {
> +		if (re_snprintf(buf, sizeof(buf), "%s,%s",
> +				sdp_media_name(st->sdpm),
> +				st->crypto_suite))
> +			(st->sess->eventh)(MENC_EVENT_SECURE, buf,
> 
> I think the paranthesis around st->sess->eventh can be removed...

Compiler does not like it:
``
 CC [M]  build-x86_64/modules/srtp/srtp.o
modules/srtp/srtp.c: In function ‘start_crypto’:
modules/srtp/srtp.c:292:5: warning: suggest explicit braces to avoid ambiguous ‘else’ [-Wparentheses]
  if (st->sess->eventh)
     ^
``
I'm also more comfortable with the parentheses, since I don't remember
which if the else belongs to.

**Comment by juha-h** [comment]
Sorry, I misunderstood which parenthesis you meant. Done,

**Comment by alfredh** [comment]
merged to master. Thanks @juha-h :)

**Pull request merged**